finalPrep/modZeroHero.js

```diff
@@ -1,52 +1,95 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
-
 // Declare two variables - heroName AND specialAbility - set to strings
+ var heroName = "Mediocre Man";
+ var specialAbility = "extreme level of mediocrity";
+
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "It's a bird, it's a plane, it's " + heroName + "!!";
+var catchphrase = `Never fear, ${heroName} is here! Prepare for an ${specialAbility}`;
+console.log(greeting);
+console.log(catchphrase);
+
 // Declare two variables - power AND energy - set to integers
+var power = 17;
+var energy = 12;
+
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = power * 500
+var fullEnergy = energy + 150
+
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = true;
+var identityConcealed = false;
+
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ['Over-Acheive-Steve', "Completed-Carl", "Sriacha-Susan"];
+var sideKicks = ['Average Alan', 'Simple Simon', 'Tony the Tiger'];
+
+
 // Print the first sidekick to your console
+console.log(sideKicks[0]);
 
 // Print the last archEnemy to the console
+console.log(sideKicks[2]);
 
 // Write some code to add a new archEnemy to the archEnemies array
+archEnemies.push("Terrible-Taylor");
 
 // Print the archEnemies array to console to ensure you added a new archEnemy
+console.log(archEnemies);
 
 // Remove the first sidekick from the sidekicks array
+sideKicks.shift();
 
 // Print the sidekicks array to console to ensure you added a new sidekick
+console.log(sideKicks);
+
 
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  var dangerLevel = dangerLevel;
+  var saveTheDay = saveTheDay;
+  var badExcuse = badExcuse;
```

Comment:
You don't need to declare these three variables. your logic below should pull dangerLevel saveTheDay, and badExcuse from the parameter.